source/partials/ctas/_marketing-consent.haml

```diff
@@ -0,0 +1,10 @@
+.marketing-consent
+  %ul.signup-cta__incentives
+    %li.signup-cta__incentive
+      %input{ type: 'radio', name: 'marketing_consent', id: 'marketing_consent_yes', value: 'yes', checked: 'checked' }
+      %label{ for: 'marketing_consent_yes' }
+        Yes, I consent to receiving Aptible marketing emails. View the <a href="/legal/privacy/" target="_blank">Aptible Privacy Statement</a>
```

Comment:
This link says "Privacy Statement", but the page it links to says "Privacy Policy" - is that intentional?